content/docs/gke/authentication.md

```diff
@@ -7,6 +7,39 @@ weight = 4
 
 ## In-cluster authentication
 
+Starting from Kubeflow version 0.6, you can consume Kubeflow from custom namespaces (i.e., namespaces other than `kubeflow`).
```

Comment:
you can consume -> "you consume"
Since we don't really allow running things anymore in the main namespace

Reply:
done